docs/src/@primer/gatsby-theme-doctocat/live-code-scope.js

```diff
@@ -52,7 +52,7 @@ export default function resolveScope(metastring) {
     ...doctocatComponents,
     ...primerComponents,
     ...(metastring.includes('drafts') ? drafts : {}),
-    ...deprecated,
+    ...(metastring.includes('deprecated') ? deprecated : {}),
```

Comment:
We will also need to add `deprecated` to the examples for deprecated components

```diff
+ ```jsx live deprecated
<BorderBox>This is a BorderBox</BorderBox>
```